lib/rushb/client.rb

```diff
@@ -1,20 +1,69 @@
+require 'rushb/resources/game'
+require 'rushb/resources/match'
+require 'rushb/resources/player'
+
 module Rushb
-  class Client < ::RocketPants::Client
+  class ClientError < StandardError; end
+
+  class Client
+    include HTTParty
 
     include Rushb::Resources::Game
-    include Rushb::Resources::Player
     include Rushb::Resources::Match
+    include Rushb::Resources::Player
 
-    version 1
-    base_uri 'http://elo.challonge.com'
+    format :json
+    base_uri "http://elo.challonge.com/1/"
```

Comment:
what's the additional `/1/` for the url for?

Reply:
@XerxesRegulus it's RushB's routing